src/github.com/mozilla-services/pushgo/simplepush/config.go

```diff
@@ -416,6 +445,21 @@ func LoadApplication(configFile ConfigFile, env envconf.Environment,
 			}
 			return h, nil
 		},
+		PluginProfile: func(app *Application) (plugin HasConfigStruct, err error) {
+			h := new(ProfileHandlers)
+			sectionName := "profile"
+			if _, ok := configFile[sectionName]; ok {
```

Comment:
This feels like it could be cleaner; we didn't really have the notion of an optional section before this.